src/ray/common/ray_config_def.h

```diff
@@ -710,3 +710,6 @@ RAY_CONFIG(int64_t, health_check_initial_delay_ms, 5000)
 RAY_CONFIG(int64_t, health_check_period_ms, 3000)
 RAY_CONFIG(int64_t, health_check_timeout_ms, 10000)
 RAY_CONFIG(int64_t, health_check_failure_threshold, 5)
+
+/// Use madvise to prevent worker coredump from including the mapped plasma pages.
+RAY_CONFIG(bool, plasma_client_madvise_dontdump, true)
```

Comment:
nit: core_dump_exclude_plasma_store?

Reply:
should we make it optional for raylet as well?

Reply:
Do we need to turn it off for the raylet as well? My impression was that if the raylet crashes, then the object store memory contents could be more significant. So best to leave on until it impacts a use case.

Reply:
I disabled it for raylet as well but removed the code

Reply:
discussed offline, I will disable the same for raylet in a separate PR